extras-pureconfig/src/main/scala/com/avast/clients/rabbitmq/extras/pureconfig/testing/testing.scala

```diff
@@ -0,0 +1,75 @@
+package com.avast.clients.rabbitmq.extras.pureconfig
+
+import cats.effect.{Effect, Resource, SyncIO}
+import com.avast.clients.rabbitmq.api._
+import com.avast.clients.rabbitmq.pureconfig.PureconfigRabbitMQConnection
+import com.avast.clients.rabbitmq.{
+  api,
+  ChannelListener,
+  ConnectionListener,
+  ConsumerListener,
+  DeliveryConverter,
+  DeliveryReadAction,
+  ProductConverter,
+  ServerChannel
+}
+import com.avast.metrics.scalaapi.Monitor
+import mainecoon.FunctorK
+
+import scala.language.higherKinds
+
+package object testing {
```

Comment:
I don't understand this module. It's called `pureconfig-extras` but it actually does conversion of effect data types. It seems strange.

Reply:
I don't like the name either (I forgot to mention it), even though it doesn't do what you're saying :smiley:

Reply:
Better toe be discussed in person.

Reply:
Solved in person